script.js

```diff
@@ -0,0 +1,395 @@
+      191: '/',
+      226: '\\',
+    },
+
+  },
+  initInput() {
+    this.elements.input = document.createElement('textarea');
+    this.elements.input.classList.add('textarea');
+    this.elements.input.id = 'use-keyboard';
+    this.elements.input.setAttribute('autofocus', 'true');
+    document.body.appendChild(this.elements.input);
+  },
+
+  initKeyboard() {
+    // create main elements
+    this.elements.main = document.createElement('div');
+    this.elements.keysContainer = document.createElement('div');
+    // setup main elements
+    this.elements.main.classList.add('keyboard');
+    this.elements.keysContainer.classList.add('keyboard--keys');
+    this.elements.keysContainer.appendChild(this.createKeys());
+    this.elements.keys = this.elements.keysContainer.querySelectorAll('.keyboard--key');
+
+    // add to DOM
+    document.body.appendChild(this.elements.main);
+    this.elements.main.appendChild(this.elements.keysContainer);
+
+    // use keybord for element whith id use-keyboard
+    const input = document.getElementById('use-keyboard');
+    this.open((currentValue) => { input.value = currentValue; });
+  },
+
+  changeLanguageWithKeyboard() {
+    document.addEventListener('keydown', (event) => {
+      if (event.shiftKey && event.keyCode === 32) {
+        this.changeLanguage();
+      }
+    });
+  },
+
+  highlightKey() {
+    document.addEventListener('keydown', (event) => {
+      document.getElementById(event.keyCode).classList.add('keyboard--key-pushed');
+    });
+
+    document.addEventListener('keyup', (event) => {
+      document.getElementById(event.keyCode).classList.remove('keyboard--key-pushed');
+      this.properties.value = document.getElementById('use-keyboard').value;
+    });
+  },
+
+  saveSelectedLanguage() {
+    sessionStorage.setItem('languageForKeyboard', this.properties.lng);
+  },
+
+  takeLanguage() {
+    this.properties.lng = sessionStorage.getItem('languageForKeyboard') ? sessionStorage.getItem('languageForKeyboard') : 'EN';
+  },
+
+  defineChar(lng, keyEventCode) {
+    if (lng === 'EN') {
+      return this.properties.alphabetEn[keyEventCode];
+    }
+    if (lng === 'RU') {
+      return this.properties.alphabetRu[keyEventCode];
+    }
+    return this;
+  },
+
+  createKeys() {
+    const fragment = document.createDocumentFragment();
+
+    // creates HTML for icons
+    const createIconHTML = (iconName) => `<i class="material-icons">${iconName}</i>`;
+
+    this.elements.keysEventCode.forEach((key) => {
+      const keyElement = document.createElement('button');
+      const insertBr = ['8', '13', '222', '191'].indexOf(key) !== -1; // includes true-false
+
+      // Add attributes/classes
+      keyElement.setAttribute('type', 'button');
+      keyElement.classList.add('keyboard--key');
+      keyElement.id = key;
+
+      switch (key) {
+        case '8':
+          keyElement.classList.add('keyboard--key_wide');
+          keyElement.innerHTML = createIconHTML('backspace');
```

Comment:
the same, use Node.removeChild and Node.appendChild because innerHTML is a high weight operation and it leads to a complete rebuild of the content of the target element